package.json

```diff
@@ -74,209 +74,209 @@
       "explorer": [
         {
           "id": "iotHubDevices",
-          "name": "Azure IoT Hub"
+          "name": "%azure-iot-toolkit.name%"
         }
       ]
     },
     "commands": [
       {
         "command": "azure-iot-toolkit.sendD2CMessage",
-        "title": "Send D2C Message to IoT Hub",
-        "category": "Azure IoT Hub"
+        "title": "%azure-iot-toolkit.sendD2CMessage.title%",
```

Comment:
I'd prefer to be `azure-iot-toolkit.command.sendD2CMessage.title` Just similar to https://github.com/microsoft/vscode-java-debug/blob/master/package.nls.json

Reply:
Okay. I will change it.